in-depth/submission_guidelines.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: docs
+title: Theme & Plugin Submission Guidelines
+headline: Theme & Plugin Submission Guidelines
+description: |
+  Submitting your Theme or Plugin for inclusion on the website is very easy.<br>
+  We've written some guidelines here to walk you through the process.
+toc:
+  as-easy-as-making-a-pull-request: As Easy as Making a Pull Request
+  the-file-layout: The File Layout
+  screenshot-specifics: Screenshot Specifics
+  metadata-breakdown: Metadata Breakdown
+  markdownyaml-template: Markdown/YAML Template
+nav-url: /docs/
+---
+
+## As Easy as Making a Pull Request
+
+Submitting content to the website is easy, all you have to do is create a new [Pull Request][] on GitHub.  To make the submission process smoother, our [Themes][] and [Plugins][] pages are dynamically generated using a collection of Markdown files.  The guidelines on this page will explain the layout and structure those files.  While we don't anticipate having to turn anyone down, we do reserve the right to reject your submission at our own discretion.  More likely though, we will work with you to get your submission up to our satisfaction.
+
+## The File Layout
+
+Pico's website is found in the [gh-pages branch][gh-pages] on GitHub.  In this branch, you'll find two folders named `_themes` and `_plugins`.  These folders contain all the individual files that make up the [Themes][] and [Plugins][] pages respectively.
+
+Each item has three types of files:
+
+* A Markdown file, containing the YAML details of your submission.
+* A `fullsize` screenshot of your item.
+  * Optionally, you can provide multiple fullsize screenshots.  These will automatically cycle on the page as a carousel.
+  * If you omit a fullsize image, the page will automatically use a two-column layout instead.  This may be useful for plugins that are too abstract to screenshot.
```

Comment:
That's not the case anymore. With `page.portfolio.defaultImage` set, that default image is shown, otherwise simply nothing. That's also true for the "jumbovision" layout (i.e. the themes page), but a screenshot is quite obligatory for a theme.